sql/src/test/java/org/opensearch/sql/sql/parser/AstBuilderTestBase.java

```diff
@@ -17,6 +17,6 @@ public class AstBuilderTestBase {
 
   protected UnresolvedPlan buildAST(String query) {
     ParseTree parseTree = parser.parse(query);
-    return parseTree.accept(new AstBuilder(query));
+    return parseTree.accept(new AstBuilder(new AstExpressionBuilder(), query));
```

Comment:
What's the purpose of changing the constructor arguments?

Reply:
Looks to be something leftover from the development of the code, the constructor arguments have been reverted to their previous arguments in https://github.com/Bit-Quill/opensearch-project-sql/pull/266/commits/b594387c580767924ca241e98137efca95259edb